addons/pinniped/config-controller/controllers/v3_controller.go

```diff
@@ -46,13 +47,13 @@ func (c *PinnipedV3Controller) SetupWithManager(manager ctrl.Manager) error {
 		).
 		Complete(c)
 	if err != nil {
-		c.Log.Error(err, "error creating pinniped config controller")
+		c.Log.Error(err, fmt.Sprintf("error creating %v", CascadeControllerV1alpha3Name))
```

Comment:
The log actually already includes the name since we set it above, maybe we could just say error creating controller here.